src/util-runmodes.c

```diff
@@ -253,8 +253,20 @@ static int RunModeSetLiveCaptureWorkersForDevice(ConfigIfaceThreadsCountFunc Mod
                               const char *live_dev, void *aconf,
                               unsigned char single_mode)
 {
+#ifdef HAVE_DPDK
+    // equal to -1 to get the first lcore id
+    int16_t dpdk_last_spawned_lcore = -1;
```

Comment:
I think I've seen `lcore_id` as uint32_t, uint16_t and int16_t. Can we settle on one type?